ote/test/clj/ote/transit_changes/detection_test_utilities.clj

```diff
@@ -0,0 +1,27 @@
+(ns ote.transit-changes.detection-test-utilities
```

Comment:
Could code formatting be fixed, some of fields are far away on the right in the file.